src/modules/functions.js

```diff
@@ -0,0 +1,41 @@
+import taskList from './taskarray.js';
+import * as element from './elements.js';
+
+// Add Tasks to Array
+export const addTask = (x, y, z) => {
+  const obj = { a: x, b: y, c: z };
+  taskList.push(obj);
```

Comment:
- [ ] OPTIONAL It will look more professional if you will use `description` or `title`, `completed `, and  `index` abbreviations instead of a, b and c
![image](https://user-images.githubusercontent.com/49064106/193232854-3f11dbdb-01f9-4d57-a5bc-8be2268e15e1.png)